app/design/frontend/Magento/storefront/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -0,0 +1,156 @@
+//
+//  Common
+//  _____________________________________________
+
+& when (@media-common = true) {
+    .page-wrapper{
+        .page-main{
+            max-width: 1920px;
+            padding-left:0;
+            padding-right:0;
+        }
+    }
+
+    .baner {
+        width: 100vw;
+        height: 75vh;
+        max-height: 928px;
+        background-color: #F3F3F3;
+        align-items: center;
+        display: flex;
```

Comment:
please check flex mixins and update:
code/vendor/magento/magento2-base/lib/web/css/source/lib/_utilities.less
from line 412